### PR TITLE
Ronald branch

### PR DESCRIPTION
**Pull request by ronaldhoang**
ignore the earlier commits it was me trying to figure out git

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -61,4 +61,6 @@ dependencies {
 
 
 
+
+
```
Comment (yoonsz25) [inline app/build.gradle]: If there's no changes, revert the file.
you can do it in terminal 
`git checkout HEAD -- <my-file.txt>`

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -50,14 +62,15 @@
...
             android:text="Open Map"
-            android:layout_marginTop="16dp"
             android:textColor="@color/white"
             android:textStyle="bold"
             tools:layout_editor_absoluteX="0dp"
             tools:layout_editor_absoluteY="360dp" />
 
     </LinearLayout>
 
+>>>>>>> 8972a911f9d3618269ead85292c7b20197b556c2
```
Comment (yoonsz25) [inline app/src/main/res/layout/activity_main.xml]: get rid of this.

**Review by yoonsz25 — CHANGES_REQUESTED** [review]

File: app/src/main/java/example/jackliu/nonewfriends/HomeActivity.java [hunk 3]
```diff
@@ -87,6 +103,7 @@ public void onAuthStateChanged(@NonNull FirebaseAuth firebaseAuth) {
                     // user auth state is changed - user is null
                     // launch login activity
                     startActivity(new Intent(HomeActivity.this, LoginActivity.class));
+                    finish();
```
Comment (yoonsz25) [inline app/src/main/java/example/jackliu/nonewfriends/HomeActivity.java:106]: @jackjunliu does this `finish()` belong here?

Comment (jackjunliu) [reply app/src/main/java/example/jackliu/nonewfriends/HomeActivity.java:106]: It does because you want to close out all pages if user is not logged in.

**Review by yoonsz25 — APPROVED** [review]

**Pull request merged**